sdks/python/apache_beam/examples/inference/anomaly_detection/write_data_to_pubsub_pipeline/config.py

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""The file defines global variables."""
+PROJECT_ID = "apache-beam-testing"
```

Comment:
As an example should this value not be unset? This way the pipeilne will fail with a clean error rather than a ACL error.

Reply:
@yeandy @damccorm What do you guys think? 
@rezarokni  I have added in the documentation to set these variables correctly before running the pipeline.

Reply:
Yeah, I think that's a good idea. We didn't do that in our previous examples, but should probably do so as well (doesn't need to happen in this PR, I can go back and do it)